tuxemon/states/combat/combat.py

```diff
@@ -663,6 +666,7 @@ def perform_action(self, user, technique, target=None):
         """
         technique.advance_round()
 
+   
```

Comment:
please avoid changing formatting unless it is more make it conforming.  PEP8 and other standards state the code should have no more than 2 newlines between code/comments.  adding this make 3 lines of whitespace, which is not conforming.